examples/step-10/step-10.cc

```diff
@@ -333,8 +333,8 @@ namespace Step10
         Triangulation<dim> triangulation;
         GridGenerator::hyper_ball(triangulation);
 
-        const MappingQ<dim> mapping(degree);
-        const FE_Q<dim>     fe(1);
+        const MappingQ<dim>   mapping(degree);
+        const FE_Nothing<dim> fe;
```

Comment:
Hm, we haven't discussed that element up until that point, whereas everyone is familiar with `FE_Q`. The actual choice of element doesn't make a difference here, so I'd just stick with `FE_Q` -- the cost is unimportant here anyway.

Reply:
I started doing as you say before, but then I found this in the piece for the area 120 lines above (the part you see here is for the perimeter):
https://github.com/dealii/dealii/blob/31ab106f152ff9a14708f08291f9937bc477bbae/examples/step-10/step-10.cc#L209-L216
I can of course revert to `FE_Q`, but I think that goes back to #6304.

Reply:
Ah, who knew? If so, then of course leaving it as is is the right way to go! Thanks for finding this!

Reply:
That tutorial was one of the first build errors I got, so my concentration level was still high :smile: